### PR TITLE
add image build for ipfs-cluster

### PR DESCRIPTION
**Pull request by zeroalphat**
Signed-off-by: zeroalphat [yotiyotiarukinopengin@gmail.com](mailto:yotiyotiarukinopengin@gmail.com)

### CODE REVIEW

**Review by zoetrope — CHANGES_REQUESTED** [review]
Please fix build failure.
You have to add TAG file.

File: ipfs-cluster/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  && cd tini \
+  && git checkout refs/tags/$TINI_VERSION \
+  && cmake . \
+  && make \
+  && chmod +x tini
+
+# Get the TLS CA certificates, they're not provided by busybox.
+
+WORKDIR $SRC_PATH
+RUN git clone https://github.com/ipfs/ipfs-cluster.git  $SRC_PATH \
```
Comment (kmdkuk) [inline ipfs-cluster/Dockerfile:41]:
```suggestion
RUN git clone https://github.com/ipfs/ipfs-cluster.git $SRC_PATH \
```

File: ipfs-cluster/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+RUN cd /tmp \
+  && git clone https://github.com/krallin/tini.git  \
+  && cd tini \
+  && git checkout refs/tags/$TINI_VERSION \
+  && cmake . \
+  && make \
+  && chmod +x tini
+
+# Get the TLS CA certificates, they're not provided by busybox.
+
```
Comment (kmdkuk) [inline ipfs-cluster/Dockerfile:39]: We don't use bybox, so I don't think this comment is necessary.

File: maintenance.md [hunk 3]
```diff
@@ -458,6 +458,7 @@ Only the base image and module dependency should be updated.
 1. Check the [releases](https://github.com/cilium/cilium/releases) page for changes.
 2. Update the `BRANCH` and `TAG` files accordingly.
 
+
```
Comment (kmdkuk) [inline maintenance.md:461]: Mistyping?

Comment (zeroalphat) [reply maintenance.md]: fixed mistyping at d0f8c997edbb1964246e31d4fbfbcb3f6e960e4e.

File: ipfs-cluster/Dockerfile (hunk 2 above)
Comment (zeroalphat) [reply ipfs-cluster/Dockerfile]: Fixed by 547d342be14e7a08942660da5d9e7149156211eb

**Review by kmdkuk — APPROVED** [review]
LGTM

**Review by zoetrope — APPROVED** [review]

**Pull request merged**